hugr_extension/src/lib.rs

```diff
@@ -81,13 +119,14 @@ mod test {
             .unwrap()
         }
 
-        let sig1 = FunctionType::new(vec![Type::UNIT, Type::UNIT], vec![Type::UNIT]);
-        let sig2 = FunctionType::new(vec![Type::UNIT], vec![Type::UNIT, Type::UNIT]);
-        let sig3 = FunctionType::new_endo(vec![Type::UNIT]);
+        let exts = ExtensionSet::from_iter([EXTENSION_ID]);
+        let sig1 = Signature::new(vec![Type::UNIT, Type::UNIT], vec![Type::UNIT]);
+        let sig2 = Signature::new(vec![Type::UNIT], vec![Type::UNIT, Type::UNIT]);
+        let sig3 = Signature::new_endo(vec![Type::UNIT]);
 
         let hole = BratOp::Hole {
             idx: 0,
-            sig: sig1.clone(),
+            sig: sig1.clone().with_extension_delta(exts.clone()),
```

Comment:
drive-by: This test was failing since the round-trip added this extension delta. Same for a few other tests below